libraries/common/components/Image/style.js

```diff
@@ -19,11 +19,19 @@ const container = (background = '#f2f2f2', paddingTop = '100%') => css({
 
 const image = css({
   position: 'absolute',
-  left: '50%',
-  maxHeight: '100%',
-  top: '50%',
-  transform: 'translate3d(-50%, -50%, 0)',
+  /*
+   * Must be here in order to keep CCP-410 fixed.
```

Comment:
Why do you tell the story of your changed css? You should do this within your commit messages what and maybe why something was changes. Additionally the ticket number references might be kind of useless for people who don't have access to our ticket system.

Reply:
Git messages are often gone when someone copy pasting the code into another folder. But fine, I will remove it.